controllers/slicegateway/slicegateway.go

```diff
@@ -331,10 +331,10 @@ func (r *SliceGwReconciler) deploymentForGatewayClient(g *kubeslicev1beta1.Slice
 	var vpnSecretDefaultMode int32 = 0644
 
 	certFileName := "openvpn_client.ovpn"
-	sidecarImg := "nexus.dev.aveshalabs.io/kubeslice/gw-sidecar:1.0.0"
+	sidecarImg := "nexus.dev.kubeslicelabs.io/kubeslice/gw-sidecar:1.0.0"
```

Comment:
I think no such repo + image exists
nexus.dev.aveshalabs.io is a nexus repo name

Reply:
Good catch! Reverting this